openedx/features/course_experience/utils.py

```diff
@@ -51,7 +51,7 @@ def recurse_mark_scored(block):
         is_scored = block.get('has_score', False) and block.get('weight', 1) > 0
         # Use a list comprehension to force the recursion over all children, rather than just stopping
         # at the first child that is scored.
-        children_scored = any(recurse_mark_scored(child) for child in block.get('children', []))
+        children_scored = any([recurse_mark_scored(child) for child in block.get('children', [])])  # pylint: disable=use-a-generator
```

Comment:
Hi there! Thanks for your contribution.

Why not fix the warning instead of disabling it?

Reply:
As you can see on this commit, this bug was introduced when someone tried to remove the warning  https://github.com/openedx/edx-platform/commit/96f0915b0fbde221dadb0b37fba6f186894260b4#diff-a7cfcadf0f9c62ab510866cfeb9e7999fccb88c50e294796409609e845e8835aR144

Reply:
Maybe we should remove the warning completely to avoid this happening again, don't you think?

Reply:
Yes, maybe a for loop?

Reply:
We could try that, it is less fancy, but at leat we could get rid of that warning.

Reply:
@mariajgrimaldi I corrected the pylint forcing the generator to be a tuple